File: app/src/main/java/com/musicseque/artist/fragments/VideoFragment.java
package com.musicseque.artist.fragments;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v7.widget.LinearLayoutManager;
import android.support.v7.widget.RecyclerView;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.musicseque.Fonts.Noyhr;
import com.musicseque.R;

import butterknife.BindView;
import butterknife.ButterKnife;

public class VideoFragment extends BaseFragment {
    View view;


    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        view = inflater.inflate(R.layout.fragment_video, null);
        ButterKnife.bind(this, view);

        initOtherViews();
        initViews();
        listeners();
        return view;
    }


    private void initOtherViews() {

    }

    private void initViews() {

        // recyclerSongs.setAdapter(new MusicAdapter(recyclerSongs,getActivity()));
    }


    private void listeners() {

    }
}

File: app/src/main/java/com/musicseque/artist/fragments/VideoFragment.kt
package com.musicseque.artist.fragments

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import butterknife.ButterKnife
import com.musicseque.R

class VideoFragment : BaseFragment() {
    var v: View? = null
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        v = inflater.inflate(R.layout.fragment_video, null)
        initOtherViews()
        initViews()
        listeners()
        return v
    }

    private fun initOtherViews() {}
    private fun initViews() { // recyclerSongs.setAdapter(new MusicAdapter(recyclerSongs,getActivity()));
    }

    private fun listeners() {}
}